### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by ivyleavedtoadflax**
Adds `remote version` command to list the current version.

### CODE REVIEW

File: remotepy/instance.py [hunk 1]
```diff
@@ -571,5 +571,72 @@ def launch(
     )
 
 
+@app.command()
+def terminate(instance_name: str = typer.Argument(None, help="Instance name")):
```
Comment (nsorros) [inline remotepy/instance.py:575]: Is this related to this PR?

**Review by nsorros — APPROVED** [review]
Looks good, a test would be good

File: remotepy/instance.py (hunk 1 above)
Comment (ivyleavedtoadflax) [reply remotepy/instance.py:575]: no, that's weird. That Pr was merged to main

**Comment by ivyleavedtoadflax** [comment]
> Looks good, a test would be good

noted, a new test suite to come in a future PR
